modules/missenaBidAdapter.js

```diff
@@ -53,6 +56,16 @@ export const spec = {
    * @return ServerRequest Info describing the request to the server.
    */
   buildRequests: function (validBidRequests, bidderRequest) {
+    const capKey = `missena.missena.capper.remove-bubble.${validBidRequests[0]?.params.apiKey}`;
+    const capping = JSON.parse(storage.getDataFromLocalStorage(capKey));
```

Comment:
What sets this in storage - do you have some other script running on the pub page? 

If it's not there this will throw an exception, I believe this is why the test run failed. (there's a `safeJSONParse` you can import from `utils.js`). 

Also, please add some tests for the new logic.

Reply:
The storage is set by the renderers we have. I will switch to `safeJSONParse` and think about a way to test this.